src/m365/aad/commands/administrativeunit/administrativeunit-get.ts

```diff
@@ -0,0 +1,121 @@
+import { AdministrativeUnit } from "@microsoft/microsoft-graph-types";
+import GlobalOptions from "../../../../GlobalOptions.js";
+import { Logger } from "../../../../cli/Logger.js";
+import { validation } from "../../../../utils/validation.js";
+import request, { CliRequestOptions } from "../../../../request.js";
+import GraphCommand from "../../../base/GraphCommand.js";
+import commands from "../../commands.js";
+import { odata } from "../../../../utils/odata.js";
+import { formatting } from "../../../../utils/formatting.js";
+import { Cli } from "../../../../cli/Cli.js";
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  id?: string;
+  displayName?: string;
+}
+
+class AadAdministrativeUnitGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ADMINISTRATIVEUNIT_GET;
+  }
+
+  public get description(): string {
+    return 'Gets information about a specific administrative unit';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        id: typeof args.options.id !== 'undefined',
+        displayName: typeof args.options.displayName !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-n, --displayName [displayName]'
```

Comment:
Let's add displayName to the this.types.strings array, to make sure numeric names don't run into issues.

Reply:
added